Source/UIBlock.h

```diff
@@ -5,11 +5,12 @@
 namespace reverb
 {
 
-    class UIBlock : public juce::Component
+    class UIBlock : public juce::GroupComponent
     {
     public:
         //==============================================================================
-        UIBlock(int numComponents, int numComponentsPerRow);
+        UIBlock(int numComponents, int numComponentsPerRow,
+            const juce::String& componentName = juce::String(), const juce::String &labelText = juce::String());
```

Comment:
This looks awkward to me, why put a default value that is an empty juce::string()?

Reply:
This is actually common practice, it lets us specify a parameter in an interface (UIBlock) that not all children need to provide but which some may need.